shack/README.md

```diff
@@ -38,25 +38,25 @@ You can stop all that with just :
 
 ### Data persistence
 
-It uses the pattern of the ["Data Volume Container"](https://docs.docker.com/userguide/dockervolumes/#creating-and-mounting-a-data-volume-container).
+This setup uses the ["Data Volume Container"](https://docs.docker.com/userguide/dockervolumes/#creating-and-mounting-a-data-volume-container) pattern.
 
-To persist, at your host-level, the inside data (proxy cached content, docker registry images) there is a command for that :
+To persist, from host-level, the container related data (proxy cached content, docker registry images), you can use the command :
```

Comment:
Effectivement, je n'arrive pas à formuler correctement cette nuance. Je
n'avais pas un bon sentiment sur cette proposition. Je vais y réfléchir.

Not sent from my PC
Le 6 mai 2015 16:32, "Damien Duportal" <notifications@github.com> a écrit :

> In shack/README.md
> <https://github.com/dduportal/rpi-utils/pull/2#discussion_r29763504>:
>
> >
> > -To persist, at your host-level, the inside data (proxy cached content, docker registry images) there is a command for that :
> > +To persist, from host-level, the container related data (proxy cached content, docker registry images), you can use the command :
>
> Pas tout à fait d'accord avec cette formulation : ce n'est pas la donnée
> du container, mais certaines données à l'intérieur du containeur.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dduportal/rpi-utils/pull/2/files#r29763504>.
>